Makefile

```diff
@@ -67,24 +67,23 @@ build-tf2-gpu:
 
 .PHONY: publish-tf1-cpu
 publish-tf1-cpu:
-	docker push $(CPU_TF1_ENVIRONMENT_NAME)-$(SHORT_GIT_HASH)
-	docker push $(CPU_TF1_ENVIRONMENT_NAME)-$(VERSION)
+	scripts/publish-docker.sh tf1-cpu $(CPU_TF1_ENVIRONMENT_NAME) $(SHORT_GIT_HASH) $(VERSION)
```

Comment:
should all of these be: `push-and-log-docker` ?

Reply:
no, but I meant to rename that script.  Fixed.